vorestation.dme

```diff
@@ -3704,5 +3704,7 @@
 #include "maps\submaps\surface_submaps\wilderness\wilderness.dm"
 #include "maps\submaps\surface_submaps\wilderness\wilderness_areas.dm"
 #include "maps\tether\tether.dm"
+#include "maps\tether\tether-03-surface3.dmm"
+#include "maps\tether\tether-08-mining.dmm"

```

Comment:
Seems like you included the tether maps in include file, which is why you're getting those Travis errors for missing areas. Those are included through defines in tether.dm, in part to avoid issues with unit tests like that.